monitoring/ceph-mixin/prometheus_alerts.libsonnet

```diff
@@ -736,5 +736,50 @@
         },
       ],
     },
+    {
+      name: 'rbdmirror',
+      rules: [
+        {
+          alert: 'CephRBDMirrorImagesPerDaemonHigh',
+          'for': '1m',
+          expr: 'sum by (ceph_daemon, namespace) (ceph_rbd_mirror_snapshot_image_snapshots) > %(CephRBDMirrorImagesPerDaemonThreshold)s' % $._config,
+          labels: { severity: 'critical', type: 'ceph_default', oid: '1.3.6.1.4.1.50495.1.2.1.10.2' },
+          annotations: {
+            summary: 'Number of image replication is now above %(CephRBDMirrorImagesPerDaemonThreshold)s' % $._config,
+            description: 'Number of image replication per daemon is not suppossed to go beyond the provided threshold',
```

Comment:
Add in the summary also the information of the threshold ... %(CephRBDMirrorImagesPerDaemonThreshold)

Reply:
Added it to the description (summary it is already there).